### ISSUE TITLE
Training parameters

### ISSUE THREAD

**Issue by jagdishbhanushali** [zc402/ctpgr-pytorch#7]
Hello,
Thank you for proving the Dataset and model. I have trained your model with default parameters, but I couldn't get good results. When I try pretrained checkpoint given from you, it is predicting correct.
Can you please let me know what parameters you used for training and what was loss (Sorry I can't read your paper properly because it's in Chinese)

Thanks,
Jagdish Bhanushali

**Comment by zc402** [comment]
Hello! for this repository you can visit an English paper at https://www.sciencedirect.com/science/article/abs/pii/S0925231219314420
I didn't change any parameters during training, I just kept training until the loss do not decrease anymore. 
Which part of the training code did not work for you? The human keypoint detection, or the gesture classification part?
